src/icons/Calendar.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+export default function Calendar() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="38.934"
+      height="41.818"
+      viewBox="0 0 38.934 41.818"
+    >
+      <g id="calendar" transform="translate(-2)">
+        <path
+          id="Path_68"
```

Comment:
If you copy svg to your project, you can remove properties like `id` and `data-name`. these are not used